skill-tree/src/main/java/com/RDS/skilltree/Endorsement/EndorsementService.java

```diff
@@ -15,4 +16,6 @@ Page<EndorsementModelFromJSON> getEndorsementsFromDummyData(
             PageRequest pageRequest, String skillID, String userID) throws IOException;
 
     EndorsementModel createEndorsement(EndorsementDRO endorsementDRO);
+
+    GenericResponse<Void> updateEndorsementStatus(String id, String status);
```

Comment:
same, Is Id is string in DB

Reply:
Updated to UUID [here](https://github.com/Real-Dev-Squad/skill-tree-backend/pull/116/commits/99349dce4a1b5c1cbe38d13899a3941e1357ffd1)